src/components/BorrowerProfile/LendCta.vue

```diff
@@ -613,6 +613,7 @@ export default {
 
 				this.$showTipMsg(this.errorMsg, 'error');
 			});
+			this.$emit('add-to-basket', { loanId: this.loanId, lendAmount: this.selectedOption });
```

Comment:
As mentioned in the ticket, this work is just to add a bubble to the BP when a user accepts an invite. So I think a lot of this code and extra querying in BP component can be removed.

We only need to show the bubble when the user accepts an invite, and it should take the form of the screenshot in the ticket.

The idea here is that it's a fix for allowing the invite process to be counted, but also show the user that they accepted an invite to a challenge. We don't want to add too much extra page load to the BP page.

Reply:
There does seem to be a lot of extra info required for this. Can we move it out of the prefetch and into it's on method within mounted instead?

Reply:
There's a good chance we don't need most of this extra info. But agreed that using mounted for this is a good approach.